src/app/dynamic/enterprise/quotas/quota-widget/component.ts

```diff
@@ -55,11 +55,14 @@ export class QuotaWidgetComponent implements OnInit, OnChanges, OnDestroy {
   @Input() showIcon = true;
   @Input() showDetailsOnHover = true;
   @Input() showEmptyPlaceholder = false;
+  @Input() isExternalCluster = false;
+  @Input() isImportedCluster = false;
 
   quotaPercentage: QuotaVariables;
   quotaDetails: QuotaDetails;
   isLoading: boolean;
   showWarning: boolean;
+  showNotApplicableText: boolean;
```

Comment:
IMO, `showNotApplicableText` i am not sure but this will be removed in future currently, ticket says 
"Quota Information Presence for External Clusters" as per my understanding just to make it consistent and let know users that Quota is not applicable to external clusters so -> may be we can update filed to let devs know what it is basically for? wdyt?


- isQuotaApplicateForExternalClusters?
- IsQuotaImplementedForExternalClusters?
- isQuotaImplemented?


any better name you can suggest? thoughts?